docs/dist-prod/open-source/renature

```diff
@@ -0,0 +1 @@
+../../dist
```

Comment:
This is a symlink

Reply:
Ah interesting, so this gets around the rewrites altogether and places everything at `/open-source/renature`? What is it symlinking to (mostly just curious)?

Reply:
There are a lot of `/ROOT_PATH` links throughout that we can't easily relativize. So when we build for production, root is `/open-source/renature`. When in dev it's `/`

So we need to account for that when previewing. The existing way for a prod build which has "root is `/open-source/renature`" assumption built in was server-side request rewrites. But this has issues like `@` getting encoded and 404-ing badge images, etc.

The simpler solution here is to build for `/open-source/renature` and then just **serve it** from `/open-source/renature` 😛 😂  and that's what this PR is...